src/components/toolbar/LineDefinition.tsx

```diff
@@ -18,44 +21,63 @@ const LineDefinition = () => {
   };
 
   return (
-    <div className="relative">
-      <Button className={`m-1 text-xl font-bold ${isMenuOpen ? "text-blue-600" : "text-sky-600"}`} variant="ghost" onClick={handleButtonClick}>
-        <span className="mr-3">&#x25BC;</span> הגדרת הישר
+    <div className=" absolute m-2 relative  rounded-md" style={isMenuOpen ? { background: "#009FDE" } : { background: "var(--cyan-100)" }}>
+      <Button
+        className={`m-4 mb-0 mr-0 w-259 text-xl font-bold ${isMenuOpen && "text-white bg-sky-100"}`}
+        variant="ghost"
+        onClick={handleButtonClick}
+        style={isMenuOpen ? { background: "#009FDE" } : { background: "var(--cyan-100)" }}
+      >
+        <img className="m-5" src={isMenuOpen ? closeMenu : openMenu} alt="open/close" />
+        <div
+          className="ml-6  text-capitalize font-Abraham text-32 font-normal "
+          style={
+            isMenuOpen
+              ? { color: "#ffffff", fontFamily: "Abraham", fontSize: "26px", fontWeight: 700 }
+              : { color: "#009FDE", fontFamily: "Abraham", fontSize: "26px", fontWeight: 700 }
+          }
+        >
+          הגדרת הישר
+        </div>
       </Button>
 
       {isMenuOpen && (
-        <div className="absolute top-8 right-0 bg-sky-100 border border-gray-300 rounded-md p-2">
+        <div className="flex flex-col items-end mt-0 rounded-tl-5 right-0  rounded-md p-2 m-2" style={{ background: "#009FDE" }}>
           <Button
-            className="block mb-2 text-blue-600 font-bold"
+            className="bg-white text-block mb-4 mt-6 hover:bg-sky-100 transition"
+            style={{ borderRadius: "20px", fontFamily: "Abraham", fontSize: "20px", fontWeight: 800 }}
             onClick={() => {
               handleMenuButtonClick(LineRange.ten);
             }}
           >
-            0 -10
+            0 - 10
           </Button>
           <Button
-            className="block mb-2 text-blue-600 font-bold"
+            className="bg-white text-block mb-4 hover:bg-sky-100 transition"
+            style={{ borderRadius: "20px", fontFamily: "Abraham", fontSize: "20px", fontWeight: 800 }}
             onClick={() => {
               handleMenuButtonClick(LineRange.twenty);
             }}
           >
             0 - 20
           </Button>
           <Button
-            className="block mb-2 text-blue-600 font-bold"
+            className="bg-white text-block mb-4 hover:bg-sky-100 transition"
+            style={{ borderRadius: "20px", fontFamily: "Abraham", fontSize: "20px", fontWeight: 800 }}
             onClick={() => {
               handleMenuButtonClick(LineRange.hundred);
             }}
```

Comment:
you can use 1 single line without {}
        onClick={() => 
              handleMenuButtonClick(LineRange.hundred)
            }